MonoDevelop.FSharpBinding/FSharpBinding.addin.xml.orig

```diff
@@ -45,33 +45,8 @@
 
   </Dependencies>
 
-  <!-- This updates the old guid to the new one on saving the project -->
-  <Extension path = "/MonoDevelop/ProjectModel/MSBuildExtensions">
-    <Class class="MonoDevelop.FSharp.CorrectGuidMSBuildExtension"/>
-  </Extension>
-
-  <!-- By default projects use the F# 3.1 targets file unless only 3.0 is available on the machine. -->
-  <!-- New projects will be created with this targets file. -->
   <Extension path="/MonoDevelop/ProjectModel/MSBuildItemTypes">
-    <!--  If FSharp 3.1 is available, use it. Note XS looks for the first DotNetProject node under 'Extension' -->
-	<Condition id="MSBuildTargetIsAvailable" target="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.1\Framework\v4.0\Microsoft.FSharp.Targets" >
-      <DotNetProject language="F#" extension="fsproj" guid="{f2a71f9b-5d33-465a-a702-920d77279786}" import="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.1\Framework\v4.0\Microsoft.FSharp.Targets"  resourceHandler="MonoDevelop.FSharp.FSharpResourceIdBuilder"/>
-	</Condition>
-
-    <!--  If FSharp 3.1 is not available, then use FSharp 3.0  -->
-    <DotNetProject language="F#" extension="fsproj" guid="{f2a71f9b-5d33-465a-a702-920d77279786}" import="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets"  resourceHandler="MonoDevelop.FSharp.FSharpResourceIdBuilder"/>
-
-    <!-- Disable this until the new project system is available
-    <Condition id="MSBuildTargetIsAvailable" target="$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.1\Framework\v4.0\Microsoft.Portable.FSharp.Targets">		
```

Comment:
No, you don't need another class. On FSharpProject you can override OnInitializeFromTemplate(). In that method you are given the Options xml element of the template, so you can check if the template is for a portable library. You can then flag the project as being a portable library (using a boolean field for example), and when OnGetDefaultImports is called, you can check that flag and return the portable imports.